libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -271,19 +283,24 @@ bool AP_CRSF_Telem::is_packet_ready(uint8_t idx, bool queue_empty)
     process_rf_mode_changes();
 
     switch (idx) {
-    case PARAMETERS:
-        // to get crossfire firmware version we send an RX device ping until we get a response
-        // but only if there are no other requests pending
-        if (_crsf_version.pending && _pending_request.frame_type == 0) {
-            if (_crsf_version.retry_count++ > CRSF_RX_DEVICE_PING_MAX_RETRY) {
-                _crsf_version.pending = false;
-                _crsf_version.minor = 0;
-                _crsf_version.major = 0;
-                gcs().send_text(MAV_SEVERITY_DEBUG,"CRSF: RX device ping failed");
-            } else {
-                _pending_request.destination = AP_RCProtocol_CRSF::CRSF_ADDRESS_CRSF_RECEIVER;
-                _pending_request.frame_type = AP_RCProtocol_CRSF::CRSF_FRAMETYPE_PARAM_DEVICE_INFO;
-                gcs().send_text(MAV_SEVERITY_DEBUG,"CRSF: requesting RX device info");
+    case PARAMETERS: {
+            const AP_RCProtocol_CRSF *crsf = AP::crsf();
+            // request RX device info only when called by the RC driver
+            if (crsf != nullptr && !crsf->is_standalone()) {
+                // to get crossfire firmware version we send an RX device ping until we get a response
+                // but only if there are no other requests pending
+                if (_crsf_version.pending && _pending_request.frame_type == 0) {
```

Comment:
is version_pending used anywhere that might matter?

Reply:
it should be shielded by the rc_options bit for crsf custom telemetry

Reply:
@andyp1per did you fix this in another PR perhaps, I might have missed it?

Reply:
okk thanks, I'm closing this one.